src/main/java/com/couchbase/lite/support/CouchbaseLiteHttpClientFactory.java

```diff
@@ -32,7 +32,7 @@
 import okhttp3.OkHttpClient;
 
 public class CouchbaseLiteHttpClientFactory implements HttpClientFactory {
-
+    private OkHttpClient client;
```

Comment:
Given the fact that each replicator creates a new factory and passes the factory around, without hugh refactory, keeping inside the factory is the best place.

Even without the fact above, keep it inside the factory seem to be a good idea. It is easier and less messy to keep the object near its source.